packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -745,6 +749,54 @@ export class DevWorkspaceClient extends WorkspaceClient {
     return workspace.metadata.annotations?.[DEVWORKSPACE_DEBUG_START_ANNOTATION] === 'true';
   }
 
+  async updateConfigData(
+    workspace: devfileApi.DevWorkspace,
+    config: api.IServerConfig,
+  ): Promise<devfileApi.DevWorkspace> {
+    const patch: api.IPatch[] = [];
+
+    const cheNamespace = config.cheNamespace;
+    if (cheNamespace) {
+      const devworkspaceConfig = `name: devworkspace-config\nnamespace: ${cheNamespace}`;
+      const devworkspaceConfigPath = `/metadata/annotations/${this.escape(
+        DEVWORKSPACE_CONFIG_ANNOTATION,
+      )}`;
+      if (workspace.metadata.annotations?.[DEVWORKSPACE_CONFIG_ANNOTATION]) {
+        if (
+          workspace.metadata.annotations?.[DEVWORKSPACE_CONFIG_ANNOTATION] !== devworkspaceConfig
+        ) {
+          patch.push({ op: 'replace', path: devworkspaceConfigPath, value: devworkspaceConfig });
+        }
+      } else {
+        patch.push({ op: 'add', path: devworkspaceConfigPath, value: devworkspaceConfig });
+      }
+    }
+
+    const currentPvcStrategy = config.defaults.pvcStrategy;
+    if (currentPvcStrategy) {
+      const devworkspaceStorageTypePath = `/metadata/annotations/${this.escape(
```

Comment:
As mentioned in my other comment, this also needs to be changed to point to the `spec.template.attributes` field. I imagine it would be  `/spec/template/attributes${this.escape(DEVWORKSPACE_STORAGE_TYPE,)}`;` though I haven't tested this suggestion.

Reply:
I have fixed it